test/docgen/view/initializer.test.ts

```diff
@@ -28,3 +28,42 @@ test.each(Language.values())('%s snapshot', (language) => {
   expect(init).toMatchSnapshot();
   expect(markdown.visitInitializer(init).render()).toMatchSnapshot();
 });
+
+const variadicTestAssembly: reflect.Assembly = Assemblies.instance.variadicExampleTest;
+const findVariadicInitializer = (): reflect.Initializer => {
+  for (const klass of variadicTestAssembly.classes) {
+    if (klass.initializer) {
```

Comment:
You are assuming here that the first initializer in the assembly has a variadic parameter. Its true now, but will it always be? 

This function needs to search for exactly what it needs, and not make any assumptions on how the assembly looks like.